docs/Developers/Writing_Good_Commits.rst

```diff
@@ -150,6 +150,10 @@ Example:
    If a specific issue may lead to an unintended behaviour from the user
    or from the program it should be considered a bug, and should be
    addresed with ``Fixes``.
+-  If an issue is labelled with ``type/bug`` you should always use ``Fixes``.
+-  Issues that require fixing some typing errors should also be referenced
+   with ``Fixes``.
+-  For all other issues use ``Closes``.
```

Comment:
Ah and here you can also remove the bullet, perfectly attaches to the previous text.